hal/armv7m/imxrt/117x/imxrt.c

```diff
@@ -389,6 +415,481 @@ __attribute__((section(".noxip"))) int _imxrt_setLevelLPCG(int clock, int level)
 }
 
 
+void _imxrt_delay(u32 ticks)
+{
+	/* TODO: use better method e.g. count cycles, must not use IRQs and peripheral timers! */
+	while (ticks-- != 0u) {
+		__asm volatile("nop");
+	}
+}
+
+
+void _imxrt_setPllBypass(u8 clk_pll, u8 enable)
+{
+	/* clang-format off */
+	switch (clk_pll) {
+		case clk_pllarm:
+			*(imxrt_common.anadig_pll + arm_pll_ctrl) = (enable != 0) ?
+				(*(imxrt_common.anadig_pll + arm_pll_ctrl) | (1uL << 17u)) :
+				(*(imxrt_common.anadig_pll + arm_pll_ctrl) & ~(1uL << 17u));
+			break;
+		case clk_pllsys1:
+			*(imxrt_common.anadig_pll + sys_pll1_ctrl) = (enable != 0) ?
+				(*(imxrt_common.anadig_pll + sys_pll1_ctrl) | (1uL << 16u)) :
+				(*(imxrt_common.anadig_pll + sys_pll1_ctrl) & ~(1uL << 16u));
+			break;
+		case clk_pllsys2:
+			*(imxrt_common.anadig_pll + sys_pll2_ctrl) = (enable != 0) ?
+				(*(imxrt_common.anadig_pll + sys_pll2_ctrl) | (1uL << 16u)) :
+				(*(imxrt_common.anadig_pll + sys_pll2_ctrl) & ~(1uL << 16u));
+			break;
+		case clk_pllsys3:
+			*(imxrt_common.anadig_pll + sys_pll3_ctrl) = (enable != 0) ?
+				(*(imxrt_common.anadig_pll + sys_pll3_ctrl) | (1uL << 16u)) :
+				(*(imxrt_common.anadig_pll + sys_pll3_ctrl) & ~(1uL << 16u));
+			break;
+		case clk_pllaudio:
+			/* TODO: access through ANATOP AI */
+			break;
+		case clk_pllvideo:
+			/* TODO: access through ANATOP AI */
+			break;
+		default:
+			break;
+	}
+	/* clang-format on */
+}
+
+
+void _imxrt_pmuBypassAnaLdo(void)
```

Comment:
```suggestion
static void _imxrt_pmuBypassAnaLdo(void)
```

Reply:
Added `static`, but `_imxrt_pmuBypassAnaLdo()` and `_imxrt_pmuBypassDigLdo()` are not currently used on evk target, will be used in second PR with PLL2, optional audio and video setup with SNVS and DCDC configuration. Added `#ifdef BYPASS_ANADIG_LDO` to omit unused warnings due to `static`.